### PR TITLE
Integration specs for views

### PR DESCRIPTION
**Pull request by leonardodiasb**
Install and configure Capybara gem.
Created integration tests for views of:
- Login page.
- User index page.
- User show page.
- Post index page.
- Post show page.

### CODE REVIEW

File: spec/features/login_spec.rb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+      click_on 'Log in'
+      expect(page).to have_content('Invalid Email or password.')
+    end
+
+    it 'renders the root page when logging in' do
+      visit '/users/sign_in/'
+      fill_in 'Email', with: 'random@test.com'
+      fill_in 'Password', with: '123456'
+      click_on 'Log in'
+      expect(page).to have_content('Signed in successfully.')
```
Comment (Davidosky007) [inline spec/features/login_spec.rb:38]: * Kindly add a Tests that redirects to the root page when a correct username and password is entered. 
* HINT 💭 `expect(current_path).to eq(root_path)`

File: spec/features/posts_index_spec.rb [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+      expect(page).to have_content('Comments: 3')
+    end
+
+    it 'I can see how many likes a post has.' do
+      expect(page).to have_content('Likes: 1')
+    end
+
+    it 'When I click on a post, it redirects me to that post show page.' do
+      find('a', text: 'Post 1').click
+      expect(page).to have_content('Post 1')
```
Comment (Davidosky007) [inline spec/features/posts_index_spec.rb]: * Kindly add the Tests that redirects to the post's show page when you click on a post. you can check HINT 💭 above ⬆️

File: spec/features/users_index_spec.rb [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    it 'I can see the number of posts each user has written.' do
+      expect(page).to have_content('Number of posts: 1')
+      expect(page).to have_content('Number of posts: 0')
+    end
+
+    it "When I click on a user, I am redirected to that user's show page." do
+      click_on 'User 1'
+      expect(page).to have_content('Bio')
+      expect(page).to have_content('Number of posts: 1')
+      expect(page).to have_content('Add Post')
```
Comment (Davidosky007) [inline spec/features/users_index_spec.rb]: * Kindly add the Tests that redirects to the user's show page when you click on a user. you can check HINT 💭 above ⬆️

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
## Hi @leonardodiasb @alonsofl 👋, 

Good job so far! 💪 
* Descriptive commit messages ✔️ 
* No linter errors ✔️ 
* Well written PR ✔️ 
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
* Kindly write tests to show redirection from one page to another  for all cases as the test [Check out this capybara cheat sheet for hints ](https://gist.github.com/zhengjia/428105)
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: spec/features/login_spec.rb (hunk 1 above)
Comment (leonardodiasb) [reply spec/features/login_spec.rb:38]: Thanks for the review and the hint !! 👍

**Review by oxenprogrammer — APPROVED** [review]
Hi @leonardodiasb, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**